server/base/src/main/java/org/apache/accumulo/server/conf/util/ZooPropEditor.java

```diff
@@ -114,7 +114,7 @@ public void execute(String[] args) throws Exception {
     } catch (Exception ex) {
       LOG.error("{} command failed", keyword(), ex);
       // hard fail - set exit status
-      System.exit(-1);
+      System.exit(1);
```

Comment:
Although the non-negative exit code is better, I don't see much value in intercepting the failure, logging it, and then exiting the process, when letting the exception fall through would basically do the same thing by default.

Reply:
Addressed in 15478a88e5